db/migrate/20200514204055_change_approved_to_string.rb

```diff
@@ -0,0 +1,5 @@
+class ChangeApprovedToString < ActiveRecord::Migration[5.1]
+  def change
+    change_column :apps, :approved, :string
```

Comment:
beacuse approved will have three states, needs to be a string instead of a boolean

Reply:
Cool, looks good